diff --git a/configure_ma35 b/configure_ma35
index 1747050b..73fd1599 100755
--- a/configure_ma35
+++ b/configure_ma35
@@ -113,9 +113,9 @@ COMMON_OPTIONS=(
 )
 
 # Extra compiler and linker flags for MA35 headers and libraries
-EXTRA_CFLAGS="-I/opt/amd/ama/ma35/include -I/opt/amd/ama/ma35/include/vpi -I/opt/amd/ama/ma35/include/xrm -I/opt/amd/ama/ma35/include/xrm_interface"
+EXTRA_CFLAGS="-I/build/source/opt/amd/ama/ma35/include -I/build/source/opt/amd/ama/ma35/include/vpi -I/build/source/opt/amd/ama/ma35/include/xma -I/build/source/opt/amd/ama/ma35/include/xrm -I/build/source/opt/amd/ama/ma35/include/xrm_interface"
 # RPATH order: First look in $PREFIX/lib (for our FFmpeg libs), then in MA35 lib (for MA35 dependencies)
-EXTRA_LDFLAGS="-L/opt/amd/ama/ma35/lib -Wl,-rpath=$PREFIX/lib:/opt/amd/ama/ma35/lib"
+EXTRA_LDFLAGS="-L/build/source/opt/amd/ama/ma35/lib -Wl,-rpath=$PREFIX/lib:/build/source/opt/amd/ama/ma35/lib"
 # Libraries needed for MA35
-EXTRA_LIBS="-lvpi -llog_ama -lxrm -lxrm_interface -lpthread -lm -ldl -lrt"
+EXTRA_LIBS="-lvpi -llog_ama -lxrm -lxrm_interface -lpthread -lm -ldl -lrt -lapr-1 -laprutil-1"
